include/rnp.h

```diff
@@ -65,6 +80,11 @@ typedef struct rnp_t {
 int rnp_init(rnp_t *);
 int rnp_end(rnp_t *);
 
+/* init, reset and free rnp operation context */
+int  rnp_ctx_init(rnp_ctx_t *);
+void rnp_ctx_reset(rnp_ctx_t *);
+void rnp_ctx_free(rnp_ctx_t *);
+
 /* debugging, reflection and information */
 int         rnp_set_debug(const char *);
```

Comment:
ahk! sorry, but I looked at non-modified lines. 
I was thinking of the spacing in the function prototype.
int         rnp_set_debug(const char *);